masonry/src/widgets/canvas.rs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2025 the Xilem Authors and the Druid Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#![warn(missing_docs)]
+
```

Comment:
I don't think the annotation is needed. Either way, we should avoid adding undocumented code.

Reply:
This must be from a copy/paste - not sure how it snuck in.